FusionIIIT/templates/ais/ais.html

```diff
@@ -0,0 +1,209 @@
+
+{% extends 'fusion/general/base.html' %}
+{% load static %}
+
+
+{% block title %}
+    Academic Information System
+{% endblock %}
+
+
+{% block body %}
+    {% block navBar %}
+        {% include 'fusion/general/../dashboard/navbar.html' %}
+    {% endblock %}
+
+
+    {% comment %}The grid starts here!{% endcomment %}
+    <div class="ui stackable doubling grid">
+
+        {% comment %}The left-margin segment!{% endcomment %}
+        <div class="column"></div>
+
+        {% comment %}
+            The left-rail segment starts here!
+        {% endcomment %}
+        <div class="three wide column">
+
+            {% comment %}The user image card starts here!{% endcomment %}
+            {% block usercard %}
+                {% include 'fusion/general/usercard.html' %}
```

Comment:
Change it accordingly. 
These URL's won't work!
Have a look at the code I pushed.
ASAP!